cbdr/cortex
# CE-612: Incorrect Carotene Code Formatting

**Purpose**
* Introduces a quick-and-dirty migration to find and replace incorrect Carotene Codes (trailing 0s were being lopped off)
* This migration reads each line of the old, incorrect Carotene CSV and uses the row's ID to match with the new, correct CSV. It then replaces the incorrect CID with the corrected one and saves.

**JIRA**
https://careerbuilder.atlassian.net/browse/CE-612

**Changes**
* Improvements and fixes
  * N/A

* Changes to developer setup/environment
  * N/A

* Architectural changes
  * N/A

* Migrations or Steps to Take on Production
  * Yes, there is a migration that finds/replaces records. Please remember to run migration!
  
* Library changes
  * N/A

* Side effects
  * N/A

**Screenshots**
* Before
N/A

* After
N/A

**Feature Server**
http://web.cortex-2.development.c66.me/

**How to Verify These Changes**
* Specific pages to visit
  * N/A

* Steps to take
  * Ensure Carotene Codes have correct CIDs/`code`s. For example, there should be a Carotene with a `code` of `49.30` that has a title `Automotive Technician`.

* Responsive considerations
  * N/A

**Relevant PRs/Dependencies**
  * N/A

**Additional Information**
N/A
